digitalidentity/service.go

```diff
@@ -113,3 +114,34 @@ func CreateShareQrCode(httpClient requests.HttpClient, sessionID string, clientS
 	err = json.Unmarshal(responseBytes, qrCode)
 	return qrCode, err
 }
+
+// Get Share QR info using the supplied sessionID
+func GetShareQr(httpClient requests.HttpClient, sessionID string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ShareSessionFetchedQrCode, err error) {
```

Comment:
Both the function and the comment use the wrong `id`. This is not the `sessionId` but the `qrId` from the qr code creation endpoint

Reply:
updated